src/go/k8s/apis/redpanda/v1alpha1/cluster_webhook.go

```diff
@@ -48,7 +48,7 @@ var (
 			corev1.ResourceCPU:    resource.MustParse("0.2"),
 		},
 	}
-	// DefaultWASMResources is default resources for coproc sidecar 
+	// DefaultWASMResources is default resources for coproc sidecar
```

Comment:
Could this change be moved to previous commit? https://github.com/vectorizedio/redpanda/pull/1948/commits/67f1f0ac2b5672721ab3d70574d12ded7c4c1eac